Frameworks/PyTorch/Getting-Started-Samples/PyTorch_HelloWorld/PyTorch_Hello_World.py

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/env python
```

Comment:
I assume python 3.x in 2020, right?

Reply:
Hi, yes, python3.x is expected.

Reply:
Nowhere in the readme or code specifies the expectation is python3, Please add references to both locations

 Example: # Python3.? and newer is required

Reply:
Hi Joe,
Users will use the exact python version from oneAPI PyTorch Conda env if they follow the steps in README.
Therefore, users don't need to worry about python version themselves, and they can find related version information in oneAPI Release notes.